templates/UserManagement/oauth_registration.html.twig

```diff
@@ -1,4 +1,4 @@
-{# needed for Google Sign #}
+{# needed for OAuth Sign in #}
 <head>
```

Comment:
have you checked if we still need this font?

Reply:
Actually, I added that font because of the google branding guidelines.